spec/system/user/user_edits_integrations_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 
 RSpec.describe "User edits their integrations", type: :system, js: true do
-  let(:user) { create(:user, saw_onboarding: true) }
```

Comment:
I've removed a bunch of these because they are redundant -- this is [the default](https://github.com/forem/forem/blob/master/spec/factories/users.rb#L19) for the user factory